osu.Game.Tests/Visual/UserInterface/TestSceneOsuHoverContainer.cs

```diff
@@ -209,8 +200,14 @@ private void doMoveOut()
 
         private sealed class OsuHoverTestContainer : OsuHoverContainer
         {
-            public Color4 HoverColourPublic => HoverColour;
-            public Color4 IdleColourPublic => IdleColour;
+            public static readonly Color4 HOVER_COLOUR = Color4.Red;
```

Comment:
It is a great idea to do it like this, what a pity I could not came up with this. 🙂